src/python/py/models/builder.py

```diff
@@ -1996,6 +2003,8 @@ def get_args():
                 enable_cuda_graph = 1 : The model can use CUDA graph capture for CUDA execution provider. If enabled, all nodes being placed on the CUDA EP
                     is the prerequisite for the CUDA graph to be used correctly. It is not guaranteed that cuda graph be enabled as it depends on the model
                     and the graph structure.
+                quant_provider = {inc, modelopt} : Quantization provider - You can choose between Intel's Neural Compressor (default) and NVIDIA's TensorRT
+                    Model Optimizer (ModelOpt). Expect slower processing but better compression when using ModelOpt.
```

Comment:
`Expect slower processing but better compression when using ModelOpt.`: remove, as are not publishing any experimental data.

Reply:
Noted and removed